browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -17,6 +17,7 @@
 #include "chrome/browser/ui/exclusive_access/exclusive_access_manager.h"
 #include "chrome/browser/ui/exclusive_access/fullscreen_controller.h"
 #include "chrome/browser/ui/tabs/tab_style.h"
+#include "chrome/browser/ui/views/frame/browser_non_client_frame_view.h"
```

Comment:
nit: seems not needed?
and let's deleted some other unused header includings such as `sidebar_button_view.h`

Reply:
Oh, yeah. Removed.